docs/themes.md

```diff
@@ -43,3 +43,12 @@ add_theme_support( 'editor-color-palette',
 ```
 
 The colors will be shown in order on the palette, and there's no limit to how many can be specified.
+
+### Disabling custom colors in block Color Palettes
+
+By default, the color pallet offered to blocks, allows the user to select a custom color different from the editor or theme default colors.
```

Comment:
Should be:

color pa*l*et**te**

Assuming that the component is properly named :)

Reply:
Nice catch, the description was updated.